src/main/java/blusunrize/immersiveengineering/api/shader/ShaderCaseDrill.java

```diff
@@ -27,9 +28,9 @@ public ShaderCaseDrill(Collection<ShaderLayer> layers)
 	}
 
 	@Override
-	public String getShaderType()
+	public ResourceLocation getShaderType()
 	{
-		return "immersiveengineering:drill";
+		return new ResourceLocation("immersiveengineering", "drill");

```

Comment:
I'd prefer `ImmersiveEngineering.MODID`, that should eliminate any possibility of typos. And the compiler should inline it, so there won't be any difference at runtime.